dax-data-set-descriptors/jfk.yaml

```diff
@@ -72,3 +72,9 @@ related_assets:
     description: Elyra example pipeline
     mime_type: text/html
     url: https://github.com/elyra-ai/examples/tree/master/pipelines/dax_noaa_weather_data
+  - name: OpenAIHub notebook
+    description: OpenAIHub exmple notebook with data mount
+    mime_type: application/x-ipynb+json
+    url: https://github.ibm.com/OpenAIHub/openaihub/blob/master/notebooks/dlf-notebooks/JFK_Data.ipynb
```

Comment:
Just to confirm, the intention is only to display the URL where the notebook can be found, not to provide programatic access to the notebook source, right?

Reply:
yes, for the time being, only the initiated can access the notebook source file

Reply:
Okay. Just wanted to make sure we weren't going to try to import it somehow using that url.